### DISCUSSION TITLE
在功能內部將其添加到收藏

### DISCUSSION THREAD

**Discussion by alctny** [he3-app/he3-feedback#60] [Ideas]
希望添加在某個具體的功能內部的時候也能將該功能添加愛到收藏。

比如當前正在使用_密碼生成器_功能，如果我想將其添加到收藏，則需要**先離開當前頁面->主頁->最近使用頁面**找到這個功能。這樣會比較麻煩。如果允許在功能內部將功能添加到收藏則可以免去這些麻煩，如果再添加 `Ctrl + D` 直接收藏該功能則會更方便。

**Comment by 1026903031** [comment]
@alctny 很好的建議 👏 ，我們之前也有考慮這方面的功能，預計會在 He3 新版本中實現 🚀

**Comment by zhenweiwang1990** [comment]
最新的版本已经支持在工具页面内部收藏工具了。

<img width="982" alt="image" src="https://user-images.githubusercontent.com/1728779/229423830-edd03806-2fdd-4dda-b51f-68e4659cd6ea.png">
